frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistFilters.tsx

```diff
@@ -124,6 +136,12 @@ export function SessionRecordingsPlaylistFilters({
                     <LemonLabel>Duration</LemonLabel>
                     {durationFilter}
                 </div>
+                {listingVersion === '3' ? (
+                    <div className="flex gap-2">
+                        <LemonLabel>Include</LemonLabel>
+                        {isActiveFilters}
+                    </div>
+                ) : null}
```

Comment:
Personally I think this should be a select box for the `durationFilter` to choose between "active" and "total" duration. To me that would feel much clearer whilst also giving the most control to the user